std_msgs/Header header
string item_id
string name
int64 confidence
float64 width
float64 height_offset
paul_vision/BBox2d box_2d
paul_vision/BBox3d box_3d #from original segmentation position
# geometry_msgs/Pose segmentation_pose
# geometry_msgs/Pose current_pose

